accurest-core/src/main/groovy/io/codearte/accurest/util/ContentUtils.groovy

```diff
@@ -0,0 +1,192 @@
+package io.codearte.accurest.util
+import groovy.json.JsonException
+import groovy.json.JsonSlurper
+import groovy.transform.TypeChecked
+import io.codearte.accurest.dsl.internal.DslProperty
+import io.codearte.accurest.dsl.internal.Headers
+import io.codearte.accurest.dsl.internal.MatchingStrategy
+import org.codehaus.groovy.runtime.GStringImpl
+
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+import static org.apache.commons.lang3.StringEscapeUtils.escapeJson
+import static org.apache.commons.lang3.StringEscapeUtils.escapeXml11
+
+@TypeChecked
+class ContentUtils {
```

Comment:
Most of it has been moved from Body and other code is tested in WiremockGroovyDslSpec